### ISSUE TITLE
after submission

### ISSUE THREAD

**Issue by sophia820** [nus-cs2030/2223-s1#15]
Hello! after submitting the lab, are we able to edit it for re submit?

**Comment by danqiye1** [comment]
Yes, you may keep uploading new versions of your code to codecrunch.

**Comment by danqiye1** [comment]
This issue is stale because it has been open for 30 days with no activity.
